documentation/tenant/master.adoc

```diff
@@ -0,0 +1,16 @@
+include::common/attributes.adoc[]
+
+= {Title}
+
+include::modules/.adoc[leveloffset=+1]
+
```

Comment:
should these just be commented out/removed (we have no modules sub directory, and obviously we wouldn't expect a doc called just ".adoc")